test/tasks/try-each-test.js

```diff
@@ -264,10 +264,12 @@ describe('tryEach', function() {
       });
 
       writeJSONFile('package.json', fixturePackage);
+      fs.closeSync(fs.openSync('yarn.lock', 'w'));
```

Comment:
hah! I just copied from Stack Overflow 😳  but I guess `writeFileSync` is newer than that answer! I’ll replace with that. 😀